turbopack/crates/turbopack/tests/node-file-trace/integration/socket.io.js

```diff
@@ -1,6 +1,6 @@
 const http = require("http");
 const io = require("socket.io");
-const opts = { port: 3000 };
+const opts = { port: 4000 + Math.floor(Math.random() * 1000) };
```

Comment:
I'm not so confident that random port allocation like this is a good idea for tests. Depending on what other tests are running, this could cause a failure.

I'd either hardcode this to a port like 4000, or use port 0. Port 0 should tell the system to allocate a high-numbered "ephemeral" port that's guaranteed to be free. On my system (`cat /proc/sys/net/ipv4/ip_local_port_range`) that's a port between 32768 and 60999.

```suggestion
const opts = { port: 0 /* request a random ephemeral port from the OS */ };
```